pom.xml

```diff
@@ -243,10 +243,10 @@
     <properties>
         <carbon.identity.package.export.project.version>${project.version}
         </carbon.identity.package.export.project.version>
-        <carbon.identity.version>5.14.0</carbon.identity.version>
-        <carbon.kernel.version>4.5.0</carbon.kernel.version>
-        <carbon.kernel.package.import.version.range>[4.5.0, 5.0.0)</carbon.kernel.package.import.version.range>
-        <carbon.identity.framework.imp.pkg.version.range>[5.14.0, 6.0.0)
+        <carbon.identity.version>5.12.153</carbon.identity.version>
+        <carbon.kernel.version>4.4.3</carbon.kernel.version>
+        <carbon.kernel.package.import.version.range>[4.4.3, 5.0.0)</carbon.kernel.package.import.version.range>
+        <carbon.identity.framework.imp.pkg.version.range>[5.12.0, 5.14.0)
```

Comment:
We have newly created a branch for the 4.4.x kernel, So can you revert these version downgrades here. https://github.com/wso2-extensions/identity-extension-utils/tree/1.0.x